Gemfile

```diff
@@ -30,7 +30,9 @@ gem 'vuejs-rails', '~> 2.3.2'
 gem 'sprockets-vue', '~> 0.1.0'
 
 gem 'rails-controller-testing'
-gem 'gdal', '~> 2.0'
+# gem 'gdal', '~> 2.0'
+# TODO Stanley - Need to change this version of gdal back to 2.0 (and remove from gitignore) when I reinstall GDAL to 2.4.4
+gem 'gdal', '~> 3.0'
```

Comment:
@stanleypliu Could you please make sure to restore this as well?

Reply:
Sorry, I'll restore it on every branch!